### PR TITLE
Test Quarkus CLI Config subcommands for creating, updating, removing and encrypting configuration properties

### PR DESCRIPTION
**Pull request by michalvavrik**
### Summary

Implements TP: https://github.com/quarkus-qe/quarkus-test-plans/pull/180

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [ ] Refactoring
- [ ] Backport
- [x] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by michalvavrik** [comment]
@gtroitsk is busy with other tasks so I think @rsvoboda will need to review

File: quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    @Test
+    public void testWrongSecretKeyHandler() {
+        // add unknown secret handler so that unit test can assert this is not reported
+        encryptBuilder.getConfigCommand()
+                .addToApplicationPropertiesFile(UNKNOWN_SECRET_HANDLER_PROPERTY, "${unknown-secret-handler::hush-hush}");
+    }
+
+    @Order(6)
+    @Test
+    public void testEncryptCommandHelp() {
```
Comment (rsvoboda) [inline quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java:162]: This looks a bit strict as help messages can change in the future. But still good to detect such changes. Fixes would be quite easy. So let's have it as is.

File: quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+                        The secret %s was encrypted to
+                        """.formatted(SECRET_4.secret));
+    }
+
+    @Order(5)
+    @Test
+    public void testWrongSecretKeyHandler() {
+        // add unknown secret handler so that unit test can assert this is not reported
+        encryptBuilder.getConfigCommand()
+                .addToApplicationPropertiesFile(UNKNOWN_SECRET_HANDLER_PROPERTY, "${unknown-secret-handler::hush-hush}");
```
Comment (rsvoboda) [inline quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java:157]: I miss an assertion in this test.

Comment (michalvavrik) [reply quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java:157]: this capability asserts that property has been added

Comment (michalvavrik) [reply quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java:157]: here it is https://github.com/quarkus-qe/quarkus-test-suite/blob/6513942134a16ee9e849a1f0278e666eaa7b416b/quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/config/surefire/EncryptPropertyTest.java#L65

Comment (michalvavrik) [reply quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java:157]: so first we assert that we added config with unknown handler, then we assert that Quarkus reports issue

File: quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java (hunk 1 above)
Comment (michalvavrik) [reply quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java:162]: yeah, it is strict, but I cannot write a test that will check that future changes are correct (there is no typo or some important message is removed) so it is easier to have a test failing and then check what is new expected value

File: quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java (hunk 2 above)
Comment (rsvoboda) [reply quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java:157]: Thanks for the details, comment on line should refer directly to `EncryptPropertyTest#testWrongSecretKeyHandler` unit test then

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java (hunk 2 above)
Comment (michalvavrik) [reply quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java:165]: done

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**